bluebottle/wallposts/models.py

```diff
@@ -209,6 +209,22 @@ class Reaction(models.Model):
     objects = ReactionManager()
     objects_with_deleted = models.Manager()
 
+    class Analytics:
+        type = 'wallpost'
+        tags = {}
+        fields = {
+            'id': 'id',
+            'user_id': 'author.id'
+        }
+
+        def extra_tags(self, instance, created):
```

Comment:
You can put this in the list of tags in the class as it won't change based on the instance.